src/main/java/com/allocadia/carbonite/CarbonRowMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.allocadia.carbonite;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+
+import lombok.SneakyThrows;
+
+public class CarbonRowMapper<T> implements RowMapper<T> {
```

Comment:
I'd probably just name everything with 'Carbonite'...really I just think 'carbonated' is kind of cute <_<